src/styles/index.js

```diff
@@ -10,12 +10,37 @@ const neutral = {
   800: "#2A2F30"
 };
 
+const fz = {
+  large: '24px',
+  medium: '20px',
+  small: '16px',
+  tiny: '12px'
+};
+
 export const mainTheme = {
+  mainFont: 'Lato, Helvetica Neue, Helvetica, sans-serif',
+  fontLarge: fz['large'],
+  fontMedium: fz['medium'],
+  fontSmall: fz['small'],
+  fontTiny: fz['tiny'],
   neutral,
   sidebarBackground: neutral[100],
   mainBackground: "white",
   colorSubheading: neutral[600]
 };
 
+/* I took this object from Auspice, not sure if we'll end up using it here
```

Comment:
Just a heads up that these styles are for _auspice_, which has a dark sidebar (and is somewhat incomplete). The one seen on nextstrain.org actually uses [these styles](https://github.com/nextstrain/nextstrain.org/blob/master/auspice/client/config.json) by utilising auspice's ability to apply customisations at build time.